### PR TITLE
july meetup

### PR DESCRIPTION
**Pull request by lorenzolfm**
@lucasdcf please add meetup link

### CODE REVIEW

File: _posts/2023-07-27-socratic-seminar-016.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+* [P2PK On Cashu](https://github.com/cashubtc/cashu/pull/270?ref=nobsbitcoin.com)
+
+## Nostr
+
+* [Nostr vending machine](https://www.nobsbitcoin.com/data-vending-machine-implementation-open-sourced/)
+
+## Outros
+
+* [SimpleX](https://github.com/simplex-chat/simplex-chat)
```
Comment (jaonoctus) [inline _posts/2023-07-27-socratic-seminar-016.md:49]: Boa! Cheguei a testar uma vez

**Comment by lucasdcf** [comment]
Link do meetup: https://www.meetup.com/pt-BR/saopaulobitdevs/events/295005078/

**Comment by lucasdcf** [comment]
Saíram as notes do LN Summit: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-July/004014.html

**Comment by brunoerg** [comment]
nao vamos mencionar que os 30 minutos iniciais sera uma apresentacao?

**Comment by lucasdcf** [comment]
Grants do OpenSats https://opensats.org/blog/bitcoin-grants-july-2023

**Comment by lucasdcf** [comment]
Splicing na Phoenix: https://acinq.co/blog/phoenix-splicing-update

File: _posts/2023-07-27-socratic-seminar-016.md (hunk 1 above)
Comment (lorenzolfm) [reply _posts/2023-07-27-socratic-seminar-016.md:50]: To la tbm, depois vou te chamar la

**Comment by lorenzolfm** [comment]
Updated with:
* meetup link
* splicing on phoenix
* opensats grants
* ln summit 2023 notes
* Mini Workshop on Bitcoin Script

**Review by lucasdcf — APPROVED** [review]

**Pull request merged**